### PR TITLE
feat: provide color icons for completion

### LINKED ISSUE #70: Add icon mapping for color completion
Color completion (kind=16) looks bland:
Screenshot 2024-01-09 at 10 54 52

compared to VS Code:
Screenshot 2024-01-09 at 10 54 15

or IJ Ultimate:
Screenshot 2024-01-09 at 11 47 54

### PR DESCRIPTION
**Pull request by fbricon**
Screenshot 2024-01-09 at 15 44 41
Screenshot 2024-01-09 at 15 45 32

Fixes #70

### CODE REVIEW

File: src/test/java/com/redhat/devtools/lsp4ij/ui/IconMapperTest.java [hunk 1]
```diff
@@ -13,20 +13,73 @@
...
-        assertNull(getIcon(null));
+        assertNull(getIcon((CompletionItemKind)null));
         for (CompletionItemKind value : CompletionItemKind.values()) {
             assertNotNull(getIcon(value), "Missing matching icon for "+value);
         }
     }
+
+    @Test
+    public void getColorIconTest()  {
+        String color = "#FFFFFF";
```
Comment (angelozerr) [inline src/test/java/com/redhat/devtools/lsp4ij/ui/IconMapperTest.java:41]: Add a test with String color = "red";

Comment (fbricon) [reply src/test/java/com/redhat/devtools/lsp4ij/ui/IconMapperTest.java:41]: won't work:
- "red" doesn't start with #
- Color.decode(key) will fail.

Is there a "color name to hex" API somewhere?

Comment (angelozerr) [reply src/test/java/com/redhat/devtools/lsp4ij/ui/IconMapperTest.java:41]: Test are hard to read, could you split the test in several methods please.

File: src/main/java/com/redhat/devtools/lsp4ij/operations/color/ColorInlayPresentation.java [hunk 2]
```diff
@@ -62,7 +63,7 @@ public void paint(@NotNull Graphics2D g, @NotNull TextAttributes textAttributes)
         g.fillRect(x, y, getSquareDimension(), getSquareDimension());
 
         // Draw a border
-        g.setColor(isDarkTheme() ? Color.WHITE : Color.BLACK);
+        g.setColor(Gray.x00.withAlpha(40));//Same border color as com.intellij.util.ui.ColorIcon
```
Comment (fbricon) [inline src/main/java/com/redhat/devtools/lsp4ij/operations/color/ColorInlayPresentation.java:66]: Makes it consistant with IJ proper and completion of color items:

Screenshot 2024-01-09 at 17 08 17

**Comment by angelozerr** [comment]
Great job @fbricon !

**Pull request merged**